### PR TITLE
GettingStarted.md: A sprinkle of minor improvements

### PR DESCRIPTION
**Pull request by AnthonyLatsis**

### CODE REVIEW

File: docs/HowToGuides/GettingStarted.md [hunk 1]
```diff
@@ -222,15 +232,7 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
 
 ### The actual build
 
-1. Make sure you have Sccache running.
-   ```sh
-   sccache --start-server
```
Comment (LucianoPAlmeida) [inline docs/HowToGuides/GettingStarted.md:227]: Is this not needed anymore?

**Review by LucianoPAlmeida — APPROVED** [review]

File: docs/HowToGuides/GettingStarted.md (hunk 1 above)
Comment (AnthonyLatsis) [reply docs/HowToGuides/GettingStarted.md:227]: `build-script` starts `sccache` for you if you pass `--sccache` (the invocation from the tutorial passes it).

Comment (AnthonyLatsis) [reply docs/HowToGuides/GettingStarted.md:227]: We only pass it in the macOS invocation though. On Linux `sccache` is presented as optional. I should probably add a sentence suggesting to add this option to the Linux invocation if the reader chose to install `sccache`.

Comment (LucianoPAlmeida) [reply docs/HowToGuides/GettingStarted.md:227]: > `build-script` starts `sccache` for you if you pass `--sccache` (the invocation in the next step passes it).

Ah ok, maybe I just couldn't find it then. To me we were only validating with `--show-stats`

> I should probably add a sentence suggesting to add this option to the Linux invocation if the reader chose to install `sccache`.

Makes sense

**Review by xedin — APPROVED** [review]

File: docs/HowToGuides/GettingStarted.md (hunk 1 above)
Comment (AnthonyLatsis) [reply docs/HowToGuides/GettingStarted.md:227]: > To me we were only validating with `--show-stats`

I only just realized what you meant here. It happens `--show-stats` starts the server implicitly, so we’re covered.

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Pull request merged**